src/data/binance/binance_client.py

```
@@ -0,0 +1,40 @@
+import os
+import logging
+from typing import Optional, Dict
+
+import requests
+
+BINANCE_API_URL = "https://api.binance.com"
+
+class BinanceClient:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)